.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,3 +1,12 @@
+---
+name: Bug Report
+about: Non critical bug reports about the Raiden Network
+---
+
+## Intro
```

Comment:
I'd suggest to put instructional parts of the issue text between html comments (i.e. `<!--` and `-->`) that way they won't show up in the rendered issue if the users forgets to remove them.

Reply:
You mean the TODO?

Reply:
No I meant the whole `Intro` section.

Reply:
Oh I see. That's a nice idea.

Reply:
Pushed the requested change. Will this work like you wanted it?